classes/hook_callbacks.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace local_pages;
+
+use html_writer;
+use moodle_url;
+
+/**
+ * Hook callbacks for local_pages.
+ *
+ * @package    local_pages
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class hook_callbacks {
+    /**
+     * Add custom head elements.
+     *
+     * @param \core\hook\output\before_standard_head_html_generation $hook
+     * @see https://github.com/praxisdigital/moodle-local_pages/blob/MOODLE_311_STABLE/lib.php
```

Comment:
Please change it to commit instead of branch.
The branch information will be change over  times when the changes is happens

Example 
```suggestion
     * @see https://github.com/praxisdigital/moodle-local_pages/blob/1cb17ca8f6fd5cb46f1faaacecda6dc84e1ed0dd/lib.php#L542-L580
```